src/main/java/it/gov/pagopa/payhub/activities/dto/fdr/FdRIngestionResponse.java

```diff
@@ -0,0 +1,18 @@
+package it.gov.pagopa.payhub.activities.dto.fdr;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+/**
+ * DTO for the FdRIngestionResponse, representing the result of file processing.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class FdRIngestionResponse {
```

Comment:
I don't like `Response`, maybe it could be `Result`?
```suggestion
public class FdRIngestionActivityResult {
```